### PR TITLE
Installation and configuration

### PR DESCRIPTION
**Pull request by deeptikeshavaprasad**
Hi,
Please review configuration.rst. It would be great to receive review comments by Tuesday, July 2nd.
Thanks,
Deepti

### CODE REVIEW

File: configuration.rst [hunk 1]
```diff
@@ -497,56 +506,62 @@ For more information about local configuration, see :ref:`Local configuration of
...
+Gauge sets the environment variables as mentioned in the ``java_ci`` environment except for ``gauge_reports_dir``.
+The value of ``gauge_reports_dir`` continues to be ``newReportsDir``.
+
+Workflow : User runs ``gauge_reports_dir=newReportsDir gauge run --env “java_ci, experimental” specs``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+``java_ci, experimental`` are the user-created environments at ``<project_root>/env``.
 
 .. note::
-   User can also explicitly set ``gauge_reports_dir=newReportsDir`` in the operating system's user shell and then run ``gauge run --env java_ci specs``
+   Alternately, user can explicitly set ``gauge_reports_dir=newReportsDir`` in the operating system's user shell and then run ``gauge run --env "java_ci, experimental" specs``
```
Comment (zabil) [inline configuration.rst]: There is a tab or space at the start of this para, is that intentional?

File: writing-specifications.rst [hunk 2]
```diff
@@ -1,40 +1,45 @@
...
+It is recommended that you use one of the following IDE plugins:
 
 .. tab-container:: ide
 
     .. tab:: Visual Studio Code
 
-        Gauge projects can be created and executed in Visual Studio Code using the `Gauge extension for VSCode <https://marketplace.visualstudio.com/items?itemName=getgauge.gauge>`__.
-        This plugin currently supports Gauge with JavaScript, Ruby and Python.
+        Gauge projects can be created and run in Visual Studio Code by using the `Gauge extension for VSCode <https://marketplace.visualstudio.com/items?itemName=getgauge.gauge>`__.
+        This plugin currently supports Gauge with JavaScript, Ruby, and Python.
```
Comment (zabil) [inline writing-specifications.rst]: we also now support Java, C# (.Net `core), Typescript so please modify this to

This plugin currently supports Gauge with Java, JavaScript, Ruby, Python, C# (.Net Core), and TypeScript.

File: writing-specifications.rst [hunk 3]
```diff
@@ -1,40 +1,45 @@
...
 
-        Gauge projects can be created and executed in IntelliJ IDEA using the `Gauge plugin for IntelliJ IDEA <https://github.com/getgauge/Intellij-Plugin/blob/master/README.md>`__.
+        Gauge projects can be created and run in IntelliJ IDEA by using the `Gauge plugin for IntelliJ IDEA <https://github.com/getgauge/Intellij-Plugin/blob/master/README.md>`__.
         This plugin only supports Gauge with Java.
 
     .. tab:: Visual Studio
 
-        Gauge projects can be created and executed in Visual Studio using the `Gauge extension for Visual Studio <https://github.com/getgauge/gauge-visualstudio/blob/master/README.md>`__.
+        Gauge projects can be created and run in Visual Studio by using the `Gauge extension for Visual Studio <https://github.com/getgauge/gauge-visualstudio/blob/master/README.md>`__.
         This plugin currently supports Gauge with C#.
```
Comment (zabil) [inline writing-specifications.rst]: Modify this line to

This plugin currently supports Gauge with C# (.Net framework).

File: writing-specifications.rst [hunk 4]
```diff
@@ -63,13 +68,11 @@ Example
 
 
 Specification Heading
-+++++++++++++++++++++
+^^^^^^^^^^^^^^^^^^^^^
 
-A specification must begin with a spec heading and a single specification can
-contain only one spec heading.
+A specification must begin with a Spec Heading. A specification contains only one Spec Heading.
```
Comment (zabil) [inline writing-specifications.rst]: > A specification contains only one Spec Heading.

With this sentence It's not clear that there is a restriction of only one Spec heading allowed per specification.

Comment (deeptikeshavaprasad) [reply writing-specifications.rst]: Okay. How about "A specification must contain only one Spec heading."?

File: configuration.rst [hunk 5]
```diff
@@ -469,14 +472,17 @@ The rules of precedence to load the environment variables are as follows:
 * Environment variables (from the environments) provided by the user in the ``--env`` flag.
 * Environment variables from ``gauge.properties`` (global configuration of Gauge; located at ``<project_root>/env``).
 * Environment variables from the ``default`` environment.
-
+  
+  .. note::
+     If the ``default`` environment is not present, Gauge internall sets the environment variables which have the same value as mentioned in ``default.properties``.
```
[the remark this answers is not in the document]
Comment (deeptikeshavaprasad) [reply configuration.rst]: Sorry, it is a typo. The word is "internally".

File: configuration.rst (hunk 1 above)
Comment (deeptikeshavaprasad) [reply configuration.rst]: Hmm.. I am not able to see anything unusual in the file. I even ran the build and viewed the web page for this file. It appears fine. I have sent screen capture. The content above is within the note.

**Pull request merged**